sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -354,9 +355,14 @@ resource
     : identifier STRING
     ;
 
+insertStatement
+    : (ctes)? insertInto queryTerm queryOrganization                                       #singleInsertQuery
+    | (ctes)? fromClause multiInsertQueryBody+                                             #multiInsertQuery
```

Comment:
I also think why we keep `insertInto` as optional in `multiInsertQueryBody` after this change, if we want to split select and insert to individual rule.

Reply:
We cannot write it like this?
```

insertStatement
    : ...
    | (ctes)? fromClause insertInto someName (insertInto? someName)      #multiInsertQuery
    ;

someName
    : querySpecification queryOrganization
    ;
```